### ISSUE TITLE
Support YAML Frontmatter

### ISSUE THREAD

**Issue by andy-cowley** [charmbracelet/glow#316]
In #146 there was an enhancement to ignore frontmatter and not render it. It would be super good if you could support simple metadata in there and make it searchable. 

`title`, `tags`, `author` are pretty common

**Comment by basebuilder** [comment]
Indeed, I second @andy-cowley suggestion. Showing frontmatter would be a dream!

I suppose it gets tricky as frontmatter can have nested objects and how to best show these :thinking:

**Comment by the-solipsist** [comment]
I use glow with my RSS feed reader (newsboat) and trafilatura to parse content. The YAML frontmatter that trafilatura outputs importantly includes author, which is important for a news article.  But glow currently can't display this.
